docs/reference-architectures/virtual-machines-linux/single-vm.md

```diff
@@ -15,25 +15,25 @@ pnp.series.prev: ./index
 
 # Run a Linux VM on Azure
 
-This reference architecture shows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single instance, and is the basis for more complex architectures such as n-tier applications. [**Deploy this solution**.](#deploy-the-solution)
+This reference architecture follows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single VM instance, and is the basis for more complex architectures such as N-tier applications. [**Deploy this solution.**](#deploy-the-solution)
 
 ![[0]][0]
 
-*Download a [Visio file][visio-download] of this architecture.*
+*Download a [Visio file][visio-download] that contains this architecture diagram.*
```

Comment:
Can we just say "Download a Visio file of this architecture diagram" ?

Reply:
The Visio file contains multiple diagrams including this one, and the tabs aren't immediately clear if you haven't used Visio much. I'd like to say something here so the user won't be confused when the file opens and the first thing the user sees is a different diagram. Thoughts?

Reply:
Makes sense.